systemtests/src/main/java/com/github/streamshub/systemtests/resourcetypes/ConsoleType.java

```diff
@@ -0,0 +1,64 @@
+package com.github.streamshub.systemtests.resourcetypes;
+
+import com.github.streamshub.console.api.v1alpha1.Console;
+import com.github.streamshub.console.api.v1alpha1.status.Condition;
+import com.github.streamshub.systemtests.utils.ResourceUtils;
+import io.fabric8.kubernetes.api.model.KubernetesResourceList;
+import io.fabric8.kubernetes.client.dsl.MixedOperation;
+import io.fabric8.kubernetes.client.dsl.NonNamespaceOperation;
+import io.fabric8.kubernetes.client.dsl.Resource;
+import io.skodjob.testframe.interfaces.ResourceType;
+
+import java.util.Optional;
+import java.util.function.Consumer;
+
+public class ConsoleType implements ResourceType<Console> {
+
+    public static MixedOperation<Console, KubernetesResourceList<Console>, Resource<Console>> consoleClient() {
+        return ResourceUtils.getKubeResourceClient(Console.class);
+    }
+
+    @Override
+    public NonNamespaceOperation<?, ?, ?> getClient() {
+        return consoleClient();
+    }
+
+    @Override
+    public String getKind() {
+        return Console.class.getSimpleName();
+    }
+
+    @Override
+    public void create(Console console) {
+        consoleClient().inNamespace(console.getMetadata().getNamespace()).resource(console).create();
```

Comment:
I believe if you use `.resource` it can find the namespace, which makes `.inNamespace` redundant since it's just taking the NS from the same object.

Reply:
Right, i think there were some cases in which it did not work, but i tried it for our resources and it seems fine without it. Thanks for noticing it.